packages/rest/src/rest.application.ts

```diff
@@ -141,23 +142,44 @@ export class RestApplication extends Application implements HttpServerLike {
    */
   route(route: RouteEntry): Binding;
 
+  /**
+   * Register a new route.
+   *
+   * ```ts
+   * function greet(name: string) {
+   *  return `hello ${name}`;
+   * }
+   * app.route('get', '/', operationSpec, greet);
+   * ```
+   */
+  route(
+    verb: string,
+    path: string,
+    spec: OperationObject,
+    handler: Function,
+  ): Binding;
```

Comment:
Please use this new API to simplify `Routing` doc page - see https://loopback.io/doc/en/lb4/Routes.html#using-partial-openapi-spec-fragments